.github/workflows/run-setup-script.yml

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
@mandarinamdar Since this is available publicly, should we document on how users can use this pipeline? To be included - What are the host requirements, steps to add this pipeline to their environment, authenticaiton, etc...

Reply:
Ideally yes. But the objective of this PR is to enable CI on every PR. We can extend this to have users use that for their deployment and testing. I am fine to include that in a separate PR

Reply:
I think for now, lets make on getting this PR to be testing only and in subsequent we can improve/document how to use it.